apps/artemis-web/components/sign-in/sign-in.tsx

```diff
@@ -82,23 +81,26 @@ const SignIn = (props) => {
 
     const res = await fetch('/api/login', {
       method: 'POST',
-      headers: { 'Content-Type': 'application/json' },
+      // withCredentials: true,
+      credentials: 'include',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
       body: JSON.stringify(body),
     });
 
     if (res.status === 200) {
       const userObj = await res.json();
-      mutate(userObj);
+
+      const cookies = new Cookies();
```

Comment:
the set cookie is coming from the server-side. why do you set it on the client-side?